src/back-end/createFiles/createUser.js

```diff
@@ -74,6 +78,8 @@ const createUser = (email, pwdHash, key, callback) => {
 	});
 }
 
+/* For exporting */
 module.exports = {
+	makeid: makeid,
```

Comment:
Don't export private helper functions to test them. You can test this by creating multiple users and checking for uniqueness of IDs.